storage_service/locations/models/wellcome.py

```diff
@@ -89,22 +89,20 @@ def move_to_storage_service(self, src_path, dest_path, dest_space):
         space_id, source_id = src_path.lstrip('/').split('/')
 
         bag = self.wellcome_client.get_bag(space_id, source_id)
-        for loc in bag['locations']:
-            if loc['provider']['id'] == 'aws-s3-ia':
-                LOGGER.debug("Fetching files from s3://%s/%s", loc['bucket'], loc['path'])
-                bucket = self.s3_resource.Bucket(loc['bucket'])
-
-                # The bag is stored unzipped (i.e. as a directory tree).
-                # Download all objects in the source directory
-                s3_prefix = loc['path'].lstrip('/')
-                objects = bucket.objects.filter(Prefix=s3_prefix)
-                for objectSummary in objects:
-                    dest_file = objectSummary.key.replace(s3_prefix, dest_path, 1)
-                    self.space.create_local_directory(dest_file)
-
-                    LOGGER.debug("Downloading %s", objectSummary.key)
-                    bucket.download_file(objectSummary.key, dest_file)
-                break
+        loc = bag['location']
+        LOGGER.debug("Fetching files from s3://%s/%s", loc['bucket'], loc['path'])
+        bucket = self.s3_resource.Bucket(loc['bucket'])
+
+        # The bag is stored unzipped (i.e. as a directory tree).
+        # Download all objects in the source directory
+        s3_prefix = loc['path'].lstrip('/')
+        objects = bucket.objects.filter(Prefix=s3_prefix)
+        for objectSummary in objects:
+            dest_file = objectSummary.key.replace(s3_prefix, dest_path, 1)
```

Comment:
(Not strictly part of this code, but I wonder if something like `os.path.relpath()` might be safer here? Only do a replace at the beginning?)

Reply:
I'll merge for now but that sounds like it's worth doing :+1: